packages/host/app/components/operator-mode/profile-info-popover.gts

```diff
@@ -0,0 +1,142 @@
+import { fn } from '@ember/helper';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+import onClickOutside from 'ember-click-outside/modifiers/on-click-outside';
+
+import { BoxelButton } from '@cardstack/boxel-ui/components';
+
+import { not } from '@cardstack/boxel-ui/helpers';
+
+import ProfileAvatarIcon from '@cardstack/host/components/operator-mode/profile-avatar-icon';
+import MatrixService from '@cardstack/host/services/matrix-service';
+
+interface Signature {
+  Args: {};
+  Element: HTMLElement;
+}
+
+export default class ProfileInfoPopover extends Component<Signature> {
+  @service declare matrixService: MatrixService;
+  @tracked private isOpened = false;
+
+  @action setPopoverProfileOpen(open: boolean) {
+    this.isOpened = open;
+  }
+
+  @action logout() {
+    this.matrixService.logout();
+  }
+
+  <template>
+    <style>
+      .profile-popover {
+        width: 280px;
+        height: 280px;
+        position: absolute;
+        bottom: 68px;
+        left: 20px;
+        z-index: 1;
+        background: white;
+        padding: var(--boxel-sp);
+        flex-direction: column;
+        border-radius: var(--boxel-border-radius);
+        box-shadow: var(--boxel-deep-box-shadow);
+        transition: all 0.1s ease-out;
+        opacity: 0;
+        display: flex;
+      }
+
+      .profile-popover.opened {
+        opacity: 1;
```

Comment:
Hiding and showing with opacity is surprising to me, is this preferable over not rendering the elements at all when it’s not shown?

These non-holistic comments are happening because I’m adding the settings view by the way 🤓

Reply:
It's because I wanted to have a slight fade in animation when opening the popover. Doing it this way does come at expense of having the popover always rendered which is not the cleanest way of doing things. I'm open to simply remove the fade in effect, or add a mechanism that adds the visual effect only after rendering the component.

Reply:
ah understood, I think we should have a good way to do that in general. In my branch I changed this so I wouldn’t have multiple selectors matching for the icon/username but I could just add parent selectors to narrow it down.